examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md

```diff
@@ -23,11 +23,11 @@ Prior to creating a ROSA STS cluster, you must create the required account-wide
         ```
         export TF_VAR_token=<ocm_offline_token>
         ```
-    3.  This value should point to your OpenShift instance.  
+    3.  This value should always point to `https://api.openshift.com` (unless you're a Red Hat engineer and you know of a different one ;) ).  
```

Comment:
That description is also copied in other README files (for example github IDP)  would you prefer to change it in all other places?

Reply:
Yes, please!

Reply:
Didn't forget about it - I'm waiting till the renaming changes will be merged